Dependency build failure because of error in `build/deps/cpp/cmake-settings/all.cmake`.

### Bug

Running `task` sometimes throws an error:
```
loading initial cache file /workspace/clp/build/deps/cpp/cmake-settings/all.cmake                                                                                                                                     
CMake Error at /workspace/clp/build/deps/cpp/cmake-settings/all.cmake:9:
  Parse error.  Expected a newline, got identifier with text "include".
```

### CLP version

41f5c3c585256b88a57379f222630d45e6de2005

### Environment

Ubuntu 22.04

### Reproduction steps

Delete `build/deps/cpp/cmake-settings/all.cmake` and rerun `task`.

--- a/components/core/CMakeLists.txt
+++ b/components/core/CMakeLists.txt
@@ -123,7 +123,7 @@ if(PROJECT_IS_TOP_LEVEL)
     # NOTE: We mark the file optional since it's not required if the user happens to have the
     # dependencies installed already.
     set(CLP_CORE_DEPS_DIR "${CMAKE_SOURCE_DIR}/../../build/deps/cpp")
-    include("${CLP_CORE_DEPS_DIR}/cmake-settings/all.cmake"
+    include("${CLP_CORE_DEPS_DIR}/cmake-settings/core-all.cmake"
             OPTIONAL
             RESULT_VARIABLE CLP_DEPS_SETTINGS_FILE_PATH
     )

--- a/components/core/CMakeLists.txt
+++ b/components/core/CMakeLists.txt
@@ -123,7 +123,7 @@ if(PROJECT_IS_TOP_LEVEL)
     # NOTE: We mark the file optional since it's not required if the user happens to have the
     # dependencies installed already.
     set(CLP_CORE_DEPS_DIR "${CMAKE_SOURCE_DIR}/../../build/deps/cpp")
-    include("${CLP_CORE_DEPS_DIR}/cmake-settings/core-all.cmake"
+    include("${CLP_CORE_DEPS_DIR}/cmake-settings/all-core.cmake"
             OPTIONAL
             RESULT_VARIABLE CLP_DEPS_SETTINGS_FILE_PATH
     )

--- a/taskfiles/deps/main.yaml
+++ b/taskfiles/deps/main.yaml
@@ -46,7 +46,7 @@ tasks:
       - task: "yscope-dev-utils:cmake:install-deps-and-generate-settings"
         vars:
           CMAKE_SETTINGS_DIR: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}"
-          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/core-all.cmake"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/all-core.cmake"
           DEP_TASK: "core-all-parallel"
       - task: "utils:combine-cpp-checksum-files"
 
@@ -106,7 +106,7 @@ tasks:
           WORK_DIR: "{{.G_SPIDER_BUILD_DIR}}"
           CMAKE_JOBS: "{{.G_CPP_MAX_PARALLELISM_PER_BUILD_TASK}}"
           CMAKE_GEN_ARGS:
-            - "-C {{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/spider-all.cmake"
+            - "-C {{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/all-spider.cmake"
             - "-DCMAKE_BUILD_TYPE=Release"
             - "-DCMAKE_INSTALL_MESSAGE=LAZY"
             - "-DSPIDER_BUILD_TESTING=OFF"
@@ -122,7 +122,7 @@ tasks:
       - task: "yscope-dev-utils:cmake:install-deps-and-generate-settings"
         vars:
           CMAKE_SETTINGS_DIR: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}"
-          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/spider-all.cmake"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/all-spider.cmake"
           DEP_TASK: "spider-all-parallel"
       - task: "utils:combine-cpp-checksum-files"
 

--- a/taskfiles/deps/main.yaml
+++ b/taskfiles/deps/main.yaml
@@ -46,6 +46,7 @@ tasks:
       - task: "yscope-dev-utils:cmake:install-deps-and-generate-settings"
         vars:
           CMAKE_SETTINGS_DIR: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/core-all.cmake"
           DEP_TASK: "core-all-parallel"
       - task: "utils:combine-cpp-checksum-files"
 
@@ -105,7 +106,7 @@ tasks:
           WORK_DIR: "{{.G_SPIDER_BUILD_DIR}}"
           CMAKE_JOBS: "{{.G_CPP_MAX_PARALLELISM_PER_BUILD_TASK}}"
           CMAKE_GEN_ARGS:
-            - "-C {{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/all.cmake"
+            - "-C {{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/spider-all.cmake"
             - "-DCMAKE_BUILD_TYPE=Release"
             - "-DCMAKE_INSTALL_MESSAGE=LAZY"
             - "-DSPIDER_BUILD_TESTING=OFF"
@@ -121,6 +122,7 @@ tasks:
       - task: "yscope-dev-utils:cmake:install-deps-and-generate-settings"
         vars:
           CMAKE_SETTINGS_DIR: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CPP_CMAKE_SETTINGS_DIR}}/spider-all.cmake"
           DEP_TASK: "spider-all-parallel"
       - task: "utils:combine-cpp-checksum-files"
 

